src/Core/Content/Media/MediaService.php

```diff
@@ -120,7 +120,7 @@ public function fetchFile(Request $request, ?string $tempFile = null): MediaFile
         }
 
         $contentType = $request->headers->get('content_type');
-        if ($contentType === 'application/json') {
+        if (strpos($contentType, 'application/json') !== false) {
```

Comment:
```suggestion
        if (strpos($contentType, 'application/json') === 0) {
```
It should start with `application/json`